catalog/core/catalog-core-camelcomponent/pom.xml

```diff
@@ -100,6 +100,10 @@
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.hazelcast</groupId>
+            <artifactId>hazelcast</artifactId>
```

Comment:
@codice/security Should we be using Hazelcast?  Most of its secure features are only documented as paid for enterprise features even though some work with the open source version of Hazelcast.  It looks like this PR is only using it for storing the file locally but eventually if we want to support this feature in a cluster configuration, we might want to consider EHCache instead (which has a cache server war that can cache over HTTP and SOAP which we can secure).